<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CDataShowDialog</class>
 <widget class="QDialog" name="CDataShowDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>576</width>
    <height>441</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>详细介绍</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,10">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget_title" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_showdata_title">
        <property name="text">
         <string>名人介绍</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>457</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QWidget" name="widget_button" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="pushButton_min">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>31</width>
             <height>23</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>31</width>
             <height>23</height>
            </size>
           </property>
           <property name="text">
            <string>➖</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_close">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>31</width>
             <height>23</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>31</width>
             <height>23</height>
            </size>
           </property>
           <property name="text">
            <string>✖</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_content" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="page_show">
         <layout class="QGridLayout" name="gridLayout_3" rowstretch="0,0,0">
          <item row="0" column="0">
           <widget class="QFrame" name="frame_pic">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>100</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>200</width>
              <height>200</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="label_person_pic">
               <property name="maximumSize">
                <size>
                 <width>200</width>
                 <height>200</height>
                </size>
               </property>
               <property name="text">
                <string>TextLabel</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="0" column="1">
           <layout class="QGridLayout" name="gridLayout_data" columnstretch="1,3">
            <item row="0" column="0">
             <widget class="QLabel" name="labelname">
              <property name="text">
               <string>姓名：</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="label_name">
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="labeldate">
              <property name="text">
               <string>时期：</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLabel" name="label_date">
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="labeltitle">
              <property name="text">
               <string>概要：</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLabel" name="label_title">
              <property name="text">
               <string>TextLabel</string>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="QLabel" name="label_describe">
            <property name="text">
             <string>    TextLabel</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QWidget" name="widget" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_readed">
               <property name="text">
                <string>已阅</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_addperson">
         <layout class="QGridLayout" name="gridLayout_2" rowstretch="0,0,0,0,0" columnstretch="0,0,0,0">
          <item row="1" column="3">
           <widget class="QLineEdit" name="lineEdit_date"/>
          </item>
          <item row="0" column="3">
           <widget class="QLineEdit" name="lineEdit_name"/>
          </item>
          <item row="2" column="2">
           <widget class="QLabel" name="labeltitle2">
            <property name="text">
             <string>概要：</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="2" column="3">
           <widget class="QTextEdit" name="textEdit_title"/>
          </item>
          <item row="3" column="0" colspan="4">
           <widget class="QTextEdit" name="textEdit_describe"/>
          </item>
          <item row="1" column="2">
           <widget class="QLabel" name="labeldate2">
            <property name="text">
             <string>时期：</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="labelname2">
            <property name="text">
             <string>姓名：</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="0" rowspan="3" colspan="2">
           <widget class="QFrame" name="frame_pic_2">
            <property name="minimumSize">
             <size>
              <width>180</width>
              <height>210</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>180</width>
              <height>210</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <property name="spacing">
              <number>10</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="label_update_pic">
               <property name="minimumSize">
                <size>
                 <width>180</width>
                 <height>180</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>180</width>
                 <height>180</height>
                </size>
               </property>
               <property name="text">
                <string>default</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_update_person_pic">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>更改图片</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="4" column="3">
           <widget class="QWidget" name="widget_2" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QPushButton" name="pushButton_delete">
               <property name="text">
                <string>删除</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_update">
               <property name="text">
                <string>修改</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>pushButton_min</tabstop>
  <tabstop>pushButton_close</tabstop>
  <tabstop>lineEdit_name</tabstop>
  <tabstop>lineEdit_date</tabstop>
  <tabstop>textEdit_title</tabstop>
  <tabstop>textEdit_describe</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_min</sender>
   <signal>clicked()</signal>
   <receiver>CDataShowDialog</receiver>
   <slot>showMinimized()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>455</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>255</x>
     <y>202</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
